cmd/gopogh/main.go

```diff
@@ -61,7 +61,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	if *dbPath != "" || os.Getenv("DB_PATH") != "" {
+	if *dbPath != "" || os.Getenv("DB_PATH") != "" || *dbBackend != "" || os.Getenv("DB_BACKEND") != "" {
```

Comment:
put this in a helper function that can read well

for example

if dbConfProvided() ...